scripts/linters/html_linter.py

```diff
@@ -341,6 +341,23 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.html_filepaths
 
+    def _get_trimmed_error_message(self, lint_messages):
+        """Remove extra bits from htmllint messages.
+
+        Args:
+            lint_messages: str. Messages returned by the html linter.
```

Comment:
because we are using third party linters and each one has a different format.

Reply:
Try to have the same function name and same param in all the methods. for consistency as well as we may have a base linter class in the future, so it would be easier to move these to the base class.